go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway v1.5.1 // indirect
 	github.com/hashicorp/consul v0.0.0-20171026175957-610f3c86a089
 	github.com/hashicorp/errwrap v1.0.0
-	github.com/hashicorp/go-azure-helpers v0.0.0-20181126135526-ec113df69f49
+	github.com/hashicorp/go-azure-helpers v0.0.0-20190129193224-166dfd221bb2
```

Comment:
is there a reason we're not pinning to the version here?

Reply:
@bflad I know, but try to run the above command yourselves - it does _not_ result in tag in `go.mod` for some reason. 🤷‍♂️

Reply:
I think what @bflad was getting at there is that `0.3.2` _does not_ start with `v`, so it doesn't look like a version tag as far as Go Modules is concerned. So it's expected behavior, per the docs.